test/protractor.conf.js

```diff
@@ -13,14 +13,13 @@ const config = {
   useAllAngular2AppRoots: true,
   specs: [ path.join(__dirname, '../e2e/**/*.e2e.ts') ],
   baseUrl: E2E_BASE_URL,
-  allScriptsTimeout: 30000,
-  getPageTimeout: 30000,
+  allScriptsTimeout: 120000,
```

Comment:
We'll only have more tests, so I don't see a future where this will go down significantly.